build/Program.fs

```diff
@@ -100,6 +100,7 @@ let generate config =
     Shell.Exec(Tools.dotnet, $"run -f {TargetFramework} -p Snowflaqe.fsproj -- --config ./{config} --generate", path [ solutionRoot; "src" ])
 
 let propsIntegration() =
+    Console.WriteLine "Generating Fable props"
```

Comment:
@Zaid-Ajaj how can we highlight the next project generation section?  Like you did with colors and ASCII graphics

Reply:
In the CLI project, I am using [this library](https://github.com/vbfox/ColoredPrintf) for colors